atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -428,6 +429,14 @@ def get_build(self, metadata, worker_metadatas):
             koji_task_owner = None
         extra['submitter'] = self.session.getLoggedInUser()['name']
 
+        resolve_comp_result = self.workflow.prebuild_results.get(PLUGIN_RESOLVE_COMPOSES_KEY)
+        if resolve_comp_result:
+            extra['image']['odcs'] = {
+                'compose_ids': resolve_comp_result['composes'],
```

Comment:
The `composes` key has a value that is a list of dicts, the results from ODCS. The compose *ID*s are available via the `id` key in each of those dicts.

So this should probably be:
```
'compose_ids': [info['id'] for info in resolve_comp_result['composes']],
```

Reply:
ahh ok will change